design/ceph/object/store.md

```diff
@@ -60,6 +60,8 @@ The pools are the backing data store for the object store and are created with s
 
 If `preservePoolsOnDelete` is set to 'true' the pools used to support the object store will remain when the object store will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
 
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. And for this option to work, `preservePoolsOnDelete` must be set to 'true'. Now the object stores can consume same pool isolated with different namespaces.
+
```

Comment:
What about the `.rgw.root` pool? Will that also be created as a rados namespace, or is that still a special pool required by rgw?

Reply:
`.rgw.root` will be automatically created when first `radosgw-admin` command is executed

Reply:
All pools need to be created by rook via CRs so that the correct replication or other settings can be applied to the pool. If radosgw-admin creates the pool, it will just use ceph defaults where rook users don't have control.